tests/Utils/BuildSchemaTest.php

```diff
@@ -713,6 +732,222 @@ enum: MyEnum
         self::assertEquals($rootFields['field2']->deprecationReason, 'Because I said so');
     }
 
+    /**
+     * it('Correctly extend scalar type').
+     */
+    public function testCorrectlyExtendScalarType(): void
```

Comment:
This does not seem to match the latest test suite from graphql-js, did you match an older version?

Reply:
https://github.com/webonyx/graphql-php/pull/928 and https://github.com/webonyx/graphql-php/pull/929 match graphql-js v15.x as v16 were still in making when I worked on them. I'll check what has changed.

Reply:
The relevant change that makes tests here and in #928 differ from the current graphql-js is https://github.com/graphql/graphql-js/commit/33c069ce64ebad1379d8c6a7937f98952ad40e63

I don't see it as something relevant for this PR. It can be solved across the whole `BuildSchemaTest` afterwards in #928.